lib/src/main/java/com/auth0/android/lock/views/ModeSelectionView.java

```diff
@@ -94,19 +96,16 @@ private void toggleBoldText(View tabView, boolean bold) {
         text.setTypeface(bold ? text.getTypeface() : null, bold ? Typeface.BOLD : Typeface.NORMAL);
     }
 
-    @Deprecated
```

Comment:
Why are removing the deprecation of these methods?

Reply:
I don't know why they were added, to begin with. The methods are not deprecated on the android SDK, otherwise, I'd receive a warning on the IDE (see screenshot). 

I assume they were marked as deprecated since they do nothing and are no longer used (see no-op). But as any "custom view" we include on this SDK, for me those are considered **internal** and shouldn't be used directly. I can't imagine someone adding a piece of our form to their app if it just won't work OOTB without the rest of the classes, for example. But still, they are public and anyone who could be calling those methods could find it breaking if we remove them.

![image](https://user-images.githubusercontent.com/3900123/82581263-ae368700-9b66-11ea-9de0-c7ff7a901d54.png)


@jimmyjames do you think I should keep the deprecated annotation? I'm OK with that as well.

Reply:
Thanks for the explanation @lbalmaceda. I think the change to remove the deprecation is fine per your reasoning 👍